pages/linux/fwconsole.md

```diff
@@ -0,0 +1,32 @@
+# fwconsole
+
+> Manage and configure your FreePBX system (PBX server).
+> More information: <https://wiki.freepbx.org>.
```

Comment:
```suggestion
> More information: <https://sangomakb.atlassian.net/wiki/spaces/FP/overview>.
```
the link you provided redirects to this site

Reply:
Or maybe use this one?
https://sangomakb.atlassian.net/wiki/spaces/PG/pages/41746505/fwconsole+commands+15

Reply:
When I was testing locally, I added this https://sangomakb.atlassian.net/wiki/spaces/PG/pages/41779247/fwconsole+commands+13
but I thought it was a long URL, so I just added the <https://wiki.freepbx.org>, which directs to the same website, ok, I will change it to https://sangomakb.atlassian.net/wiki/spaces/PG/pages/41779247/fwconsole+commands+13